client-js/src/client.ts

```diff
@@ -7,15 +6,15 @@ import RecordsClient from './clients/RecordsClient';
 import EfficiencyClient from './clients/EfficiencyClient';
 import NameChangesClient from './clients/NameChangesClient';
 import CompetitionsClient from './clients/CompetitionsClient';
-import BaseAPIClient from './clients/BaseAPIClient';
+import HttpClient from './clients/HttpClient';
 
 interface WOMClientOptions {
   apiKey?: string;
   userAgent?: string;
   baseAPIUrl?: string;
 }
 
-export default class WOMClient extends BaseAPIClient {
+export default class WOMClient {
```

Comment:
WOMClient extended BaseAPIClient so that I could still use the base http client without needing to be tied to an existing "module",

for example, for some undocumented endpoints like `/stats`, I could just do

`new WOMClient().getRequest("/stats")`, or an actual example of this usage:

https://github.com/wise-old-man/discord-bot/blob/35242eb14ce314ffc7ae7aefb5caa5afec1cc63c/src/services/wiseoldman.ts#L160

Reply:
That's fair, I'll rework it to continue providing that functionality.

Reply:
Ok, all that code in the example file should continue running as is with no changes necessary now.